Modules/ip.py

```diff
@@ -0,0 +1,42 @@
+import main
+import os
+from requests import get
+import time
+
+def Init(module: main.Module):
+    module.color = '1010101'
+
+def Update(module: main.Module):
+    # Check if not a timer for the module is set
+    ip_path = "module_data/ip/timer.txt"
+    last_ip_path = "module_data/ip/last_ip.txt"
+    if not os.path.exists(ip_path):
+        os.makedirs(os.path.dirname(ip_path), exist_ok=True)
+        with open(ip_path, 'w') as file:
+            file.write(str(time.time()))
+    elif not os.path.exists(last_ip_path):
+        os.makedirs(os.path.dirname(last_ip_path), exist_ok=True)
+        with open(last_ip_path, 'w') as file2:
+            try:
+                file2.write(get('https://api64.ipify.org', timeout=2).text) 
```

Comment:
There is a python module for this: https://pypi.org/project/ipify/

Reply:
In case this module will not be used, implement proper error handling using [response.raise_for_status()](https://3.python-requests.org/user/quickstart/). Also, timeout=2 seems a bit short

Reply:
The ipify libary uses some since python 3.8 deprecated functions and therefore doesnt work. Error ahndeling is now implemented.